asv_bench/benchmarks/io/json.py

```diff
@@ -82,6 +84,7 @@ def setup(self, orient, frame):
         timedeltas = timedelta_range(start=1, periods=N, freq="s")
         datetimes = date_range(start=1, periods=N, freq="s")
         ints = np.random.randint(100000000, size=N)
+        longints = sys.maxsize * np.random.randint(100000000, size=N)
```

Comment:
@jreback Not sure where we want the `longints` to go - whether into `ints` or separately. I would think we also want to make sure we have both positive and negative long ints in there?

Reply:
this prob fine